tests/augment/en/test_rephrase.py

```diff
@@ -7,41 +7,67 @@
 from openai.types.chat import ChatCompletion
 
 from langcheck.augment.en import rephrase
+from langcheck.metrics.eval_clients import (
+    AzureOpenAIEvalClient,
+    OpenAIEvalClient,
+)
 
 
 @pytest.mark.parametrize(
     "instances, num_perturbations, expected",
     [
-        ("List three representative testing methods for LLMs.", 1,
-         ["Identify three typical methods used for evaluating LLMs."]),
-        (["List three representative testing methods for LLMs."], 2, [
-            "Identify three typical methods used for evaluating LLMs.",
-            "Identify three typical methods used for evaluating LLMs."
-        ]),
+        (
+            "List three representative testing methods for LLMs.",
+            1,
+            ["Illuminate three representative methods for testing LLMs."],
+        ),
+        (
+            ["List three representative testing methods for LLMs."],
+            2,
+            [
+                "Illuminate three representative methods for testing LLMs.",
+                "Identify three typical techniques for testing LLMs.",
+            ],
+        ),
     ],
 )
-def test_rephrase(instances: list[str] | str, num_perturbations: int,
-                  expected: list[str]):
+def test_rephrase(
+    instances: list[str] | str, num_perturbations: int, expected: list[str]
+):
     mock_chat_completion = Mock(spec=ChatCompletion)
     mock_chat_completion.choices = [
-        Mock(message=Mock(
-            content="Identify three typical methods used for evaluating LLMs."))
+        Mock(
+            message=Mock(
+                content="Identify three typical methods used for evaluating LLMs."
+            )
+        )
```

Comment:
How should I modify here? It's now mocking the openai api return value as a fixed sentence is expected, but can I do the same for different outputs?

Reply:
I did the same for multiple augmentations (`chat.Completions.create` returns the elements in `side_effect` sequentially)

Reply:
Sounds good!!